test/e2e/framework/pods.go

```diff
@@ -29,70 +29,74 @@ import (
 	. "github.com/onsi/gomega"
 )
 
-// TODO: Consolidate pod-specific framework functions here.
-
 // Convenience method for getting a pod client interface in the framework's namespace.
-func (f *Framework) PodClient() unversioned.PodInterface {
-	return f.Client.Pods(f.Namespace.Name)
+func (f *Framework) PodClient() *PodClient {
```

Comment:
This function is intended to replace `f.Client.Pods(...)`, and we always use `podClient := f.Client.Pods(...)` in the test, so I name it as `PodClient`.